### PR TITLE
[WIP] - Hackathon Events - Sponsor Profiles addition

### PR DESCRIPTION
**Pull request by androolloyd**
##### Description

- new column added, sponsor_profiles, many to many relationship with Profile, limited to only profiles that have a data__type=Organization
the org data returned for the hackathon explorer has been updated to use this new relationship

##### Refers/Fixes

Ref #6197 
##### Testing

Coming soon

Image 2020-03-17 at 8 18 38 PM

Image 2020-03-17 at 8 19 18 PM

### CODE REVIEW

File: app/dashboard/views.py [hunk 1]
```diff
@@ -3631,18 +3631,15 @@ def hackathon(request, hackathon=''):
     if timezone.now() < hackathon_event.start_date and not request.user.is_staff:
         return redirect(reverse('hackathon_onboard', args=(hackathon_event.slug,)))
 
-    # TODO: Refactor post orgs
     orgs = []
-    for bounty in Bounty.objects.filter(event=hackathon_event, network=network).current():
+    for sponsor_profile in hackathon_event.sponsor_profiles.all():
```
Comment (octavioamu) [inline app/dashboard/views.py]: probably we don't need to do this for anymore, just a "for" on the template

File: app/dashboard/migrations/0092_auto_20200318_0001.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.4 on 2020-03-18 00:01
+
+import django.contrib.postgres.fields
```
Comment (octavioamu) [inline app/dashboard/migrations/0092_auto_20200318_0001.py]: can we rebase and recreate this migration? now 0092 is already used by other migration

File: app/chat/tasks.py [hunk 3]
```diff
@@ -206,12 +204,11 @@ def create_channel(self, options, bounty_id=None, retry: bool = True) -> None:
...
             return new_channel
         except ConnectionError as exc:
-            logger.info(str(exc))
-            logger.info("Retrying connection")
+            logger.debug(str(exc))
             self.retry(30)
         except Exception as e:
             print("we got an exception when creating a channel")
-            logger.error(str(e))
+            logger.debug(str(e))
```
Comment (octavioamu) [inline app/chat/tasks.py]: are this changes on this file related?

File: app/dashboard/migrations/0092_auto_20200318_0001.py (hunk 2 above)
Comment (androolloyd) [reply app/dashboard/migrations/0092_auto_20200318_0001.py:3]: sure

File: app/chat/tasks.py (hunk 3 above)
Comment (androolloyd) [reply app/chat/tasks.py:211]: trailing, will remove.

**Comment by octavioamu** [comment]
I believe we need this soon is some blocker here?

**Pull request closed without merging**